geps/gep-3567/metadata.yaml

```diff
@@ -11,23 +11,23 @@ relationships:
   # obsoletes indicates that a GEP makes the linked GEP obsolete, and completely
   # replaces that GEP. The obsoleted GEP MUST have its obsoletedBy field
   # set back to this GEP, and MUST be moved to Declined.
-  obsoletes: {}
-  obsoletedBy: {}
+  obsoletes: []
+  obsoletedBy: []
   # extends indicates that a GEP extends the linked GEP, adding more detail
   # or additional implementation. The extended GEP MUST have its extendedBy
   # field set back to this GEP.
   extends: {}
```

Comment:
Should also `extends` and `extendedBy` be refactored to be a list insetad of an object?

Reply:
hum probably, the weird thing is that the yaml parser didn't complained about it... let me see what did I missed

Reply:
@snorwin found it! the json marker on the GEP API is wrong!

```
	// The GEP Updates the listed GEPs.
	Extends []GEPRelationship `json:"updates,omitempty"`
	// The GEP is Updated by the listed GEPs.
	ExtendedBy []GEPRelationship `json:"updatedBy,omitempty"`
```
It is called Extends and ExtendedBy but the markers are "updates" and "updatedBy".

@robscott How do we want to proceed here? This is not really a big deal unless you are doing some parsing of metadata.yaml as I am :)

Reply:
We should fix this, I probably just forgot to update the json tag when we changed the name.

Reply:
yeah +1, worth fixing this